scripts/lua/routing.lua

```diff
@@ -89,8 +90,15 @@ function _M.findRedisKey(resourceKeys, tenant, path)
   -- Check for exact match or case where resource is "/"
   local redisKey = utils.concatStrings({"resources:", tenant, ":", path})
   local redisKeyWithSlash = utils.concatStrings({redisKey, "/"})
+  -- Check for x-cf-forwarded-url
+  local cfRedisKey
+  local cfUrl = ngx.req.get_headers()["x-cf-forwarded-url"]
```

Comment:
Same comment applies here. We shouldn't be adding cf-specific things to general routing. The policy needs to be able to alter the upstream URL and have everything handled there.